### PR TITLE
Finalize Documentation

### PR DESCRIPTION
**Pull request by iteles**
Couldn't see an open pull requestt documentation so created this one for ease of review. 

Related to #411

### CODE REVIEW

File: docs/home.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Candidates:
+
+  ```js
+  [ { listFavourite: [],
+      favourite: false,
+      id: 'some id',
+      fullname: 'CandidateCSV',
+      firstName: 'CandidateCSV',
+      headline: '',
+      current: '',
```
Comment (iteles) [inline docs/home.md]: If this is an example data set, it would be good to have some example data in all these empty fields please :)

Comment (heron2014) [inline docs/home.md]: @iteles I have updated the candidate object with more information.

**Comment by iteles** [comment]
Thanks @heron2014 !

@nelsonic Could you please cast an eye over this and merge?

**Comment by nelsonic** [comment]
Nice one @heron2014, @SimonLab & @iteles ! 🎉 
(_future dev will thank you for this..._)

**Pull request merged**